src/main/java/ch/jalu/configme/properties/EnumSetProperty.java

```diff
@@ -0,0 +1,31 @@
+package ch.jalu.configme.properties;
+
+import ch.jalu.configme.properties.types.EnumSetPropertyType;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * Introduce an EnumSetProperty that goes along with EnumSetPropertyType.
+ *
+ * @param <E> enum
+ */
+public class EnumSetProperty<E extends Enum<E>> extends SetProperty<Set<E>> {
```

Comment:
`SetProperty` is defined like this:

```java
public class SetProperty<E> extends TypeBasedProperty<Set<E>> {
```

So extending `SetProperty`  as `SetProperty<Set<E>>` means we end up having a `Property<Set<Set<E>>`, which is not what we want. We want to extend `SetProperty<E>` so that the property's value is `Set<E>`.